ci/script_multi.sh

```diff
@@ -6,6 +6,7 @@ source activate pandas
 
 if [ -n "$LOCALE_OVERRIDE" ]; then
     export LC_ALL="$LOCALE_OVERRIDE";
+    export LANG="$LOCALE_OVERRIDE";
```

Comment:
We have this in `ci/script_single.sh` but not here. I don't see why in one case should be needed and not in the other. I guess it's missing here (or could be removed there).

Reply:
you could add this would prob be ok
but all of the locale tests are done in the single i think